packages/appstore/src/components/modals/wallets-migration-failed/wallets-migration-failed.tsx

```diff
@@ -0,0 +1,115 @@
+                            <div ref={wallets_migration_failed_ref}>
+                                <Modal.Body>
+                                    <Text
+                                        as='h1'
+                                        size='s'
+                                        color='prominent'
+                                        weight='bold'
+                                        className='wallets-migration-failed__title'
+                                    >
+                                        {localize('Sorry for the interruption')}
+                                    </Text>
+                                    <Text size='xs'>
+                                        {localize(
+                                            "We're unable to complete with the Wallet upgrade. Please try again later or contact us via live chat."
+                                        )}
+                                    </Text>
+                                </Modal.Body>
+                                <Modal.Footer className='wallets-migration-failed__footer'>
+                                    <Button secondary large onClick={handLiveChatButtonClick}>
+                                        {localize('Go to live chat')}
+                                    </Button>
+                                    <Button primary large onClick={handleClose}>
+                                        {localize('Back to Trader’s Hub')}
+                                    </Button>
+                                </Modal.Footer>
+                            </div>
+                        </Modal>
+                    </DesktopWrapper>
+                    <MobileWrapper>
+                        <Modal
+                            is_open={is_wallet_migration_failed}
+                            toggleModal={handleClose}
+                            width='32.3rem'
+                            should_header_stick_body={false}
+                            has_close_icon={false}
+                        >
+                            <div ref={wallets_migration_failed_ref}>
+                                <Modal.Body className='wallets-migration-failed--mobile'>
+                                    <Text
+                                        as='h1'
+                                        size='xs'
+                                        color='prominent'
+                                        weight='bold'
+                                        className='wallets-migration-failed__title--mobile'
+                                    >
+                                        {localize('Sorry for the interruption')}
+                                    </Text>
+                                    <Text size='xxs'>
+                                        {localize(
+                                            "We're unable to complete with the Wallet upgrade. Please try again later or contact us via live chat."
```

Comment:
I think you missed on the EU scenario, the text is different.

Reply:
@aizad-deriv Can you give me the link because these 3 links

(EU) https://www.figma.com/file/VDaEitWKa2OoGQYXVA4jZW/Working-file---%5BWallets%5D-EU-Trader%E2%80%99s-hub?type=design&node-id=434-298763&t=93UK7oautZVbcQ0E-0

(Low-risk) https://www.figma.com/file/ZMPTjqYxLsxH8Elb90sQ5Q/Working-file---%5BWallets%5D-Low-risk-ROW-Trader%E2%80%99s-hub?type=design&node-id=36669-570271&t=tV3D7PaISuW1ODAf-0

(High-risk) https://www.figma.com/file/wa354FXfsiX8ay81GKwNQf/Working-file---%5BWallets%5D-High-risk-ROW-Trader%E2%80%99s-hub?type=design&node-id=322-259261&t=JX0HRoiyh0fTj8vA-0

The text are equal for all jurisdictions for me

Reply:
Sorry my bad I look at this modal 🤣
https://www.figma.com/file/wa354FXfsiX8ay81GKwNQf/Working-file---%5BWallets%5D-High-risk-ROW-Trader%E2%80%99s-hub?type=design&node-id=322-284966&t=66slmXW3gcEorSoc-4